src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -511,3 +513,19 @@
     (:less-than-three-minutes constants/wallet-transaction-estimation) "1-3"
     (:less-than-five-minutes constants/wallet-transaction-estimation)  "3-5"
     ">5"))
+
+(defn add-missing-balance-per-chain-values
+  "Adds any missing chain balance (0) to the balance-per-chain map based on token supported chains as
+   status-go returns balance for that chain only if the balance is positive."
+  [{:keys [balances-per-chain token-supported-chains]}]
+  (reduce
+   (fn [result chain-id]
+     (if (contains? result chain-id)
+       result
+       (assoc result
+              chain-id
+              {:chain-id    chain-id
+               :raw-balance (money/->bignumber 0)
+               :balance     "0"})))
```

Comment:
nit: since this map is constant, I'd create it outside of this `reduce`, in a `let`.